powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -562,12 +566,37 @@ public Response updateIos(String appId, String bundle, String keyId, String team
         return response;
     }
 
+    /**
+     * Update APNs details for an application credentials entity.
+     *
+     * @param appId ID of application credentials entity.
+     * @param bundle The APNs bundle record.
+     * @param keyId The APNs key ID record.
+     * @param teamId The APNs team ID record.
+     * @param environment The APNs environment.
+     * @param privateKey The APNs private key bytes.
+     * @return Response from server.
+     * @throws PushServerClientException Thrown when communication with Push Server fails.
+     */
+    public Response updateApns(String appId, String bundle, String keyId, String teamId, ApnsEnvironment environment, byte[] privateKey) throws PushServerClientException {
+        final String privateKeyBase64 = Base64.getEncoder().encodeToString(privateKey);
+        final UpdateApnsRequest request = new UpdateApnsRequest(appId, bundle, keyId, teamId, environment, privateKeyBase64);
```

Comment:
May we replace many arguments with a single request object? The objection is related to the constructor, which has too many parameters of the same type (it could be replaced with a builder).